### ISSUE TITLE
There is no GPS option available thru the setting page

### ISSUE THREAD

**Issue by lutin11** [ubports/ubuntu-touch#1512]
Device: nexus 5
Chanel: devel
Os version: 2020-07-09/2

How to reproduce:
Go to setting panel, there in no place to set GPS settings

**Comment by Flohack74** [comment]
Which settings would you expect here?

**Comment by lutin11** [comment]
> Which settings would you expect here?
I made a mistake in the bug report title, I should have said" There is no 'Location' options available threw the setting page.
I thought about:
turn On/Off
Localisation done threw GPS or WIFI
List of apps that are allowed to use the localisation.

**Comment by Arc676** [comment]
I guess it's not obvious but you can find GPS settings under Privacy > Location. There, you can enable and disable GPS, as well as control which apps have access to your location. You can also control this under Privacy > App Permissions > Location. Otherwise you can turn enable and disable GPS from the menu bar (it's between Bluetooth and data transfer). There's also a shortcut to GPS privacy settings. There's no localization by WLAN though.

GPS doesn't appear in the menu bar when it's disabled so finding it means the user has to manually scroll through the menus to find Location. There is, however, a search bar in the settings app. But (this might be content for a separate issue) I can't find GPS settings by searching for it. My Nexus 5 is set to German and I tried searching for "App Berechtigungen", "Standort", "GPS", and for good measure also "Location" and "Position" in English but I got no results. So the problem is more the discoverability of the relevant settings, not the available settings themselves.

**Comment by lutin11** [comment]
> I guess it's not obvious but you can find GPS settings under Privacy > Location. There, you can enable and disable GPS, as well as control which apps have access to your location. You can also control this under Privacy > App Permissions > Location. Otherwise you can enable and disable GPS from the menu bar (it's between Bluetooth and data transfer). There's also a shortcut to GPS privacy settings in the menu. There's no localization by WLAN though.
>
That's true but as you said it's not obvious ;) I would expect to have a localisation icon directly into the main settings panel.
> 
> GPS doesn't appear in the menu bar when it's disabled so finding it means the user has to manually scroll through the menus to find Location. There is, however, a search bar in the settings app. But (this might be content for a separate issue) I can't find GPS settings by searching for it. My Nexus 5 is set to German and I tried searching for "App Berechtigungen", "Standort", "GPS", and for good measure also "Location" and "Position" in English but I got no results. So the problem is more the discoverability of the relevant settings, not the available settings themselves.

**Comment by UniversalSuperBox** [comment]
I've checked on a couple of Android devices I have. One has "Location" as a single top-level settings page, another has it as a submenu under "Security". For this reason, I don't think the location of the setting is strictly wrong. It is possible that a new Location page could be created as a top-level item, if someone were so inclined. That would involve creating a new Settings plugin.

It would also be possible to make it so searches for things like "location" or "gps" found the security and privacy page. This would be done by adding more keywords to the `security-privacy.settings` keywords list: https://github.com/ubports/system-settings/blob/54ac6bfc292228395d42618c40e96f102ed9a6f4/plugins/security-privacy/security-privacy.settings

If either of these sound like acceptable ways forward to you, I can create a new issue requesting that on the system-settings repository and close this report.
